magento/pwa-studio
# [Doc] Update README file

## Description

Updates the README file with more up to date information about the project and recent contributors.

## Related Issue

Closes #1697 

## Verification Steps

1. Open README.md
2. Verify content about the project is accurate
